src/connector/mssql/error.rs

```diff
@@ -97,6 +97,19 @@ impl From<tiberius::error::Error> for Error {
 
                 builder.build()
             }
+            tiberius::error::Error::Server(e) if e.code() == 1505 => {
+                let mut splitted = e.message().split('\'');
+                let index = splitted.nth(3).unwrap().to_string();
```

Comment:
When we have time to tackle this, we should really think of a more robust approach for error metadata extraction :/